src/components/landing.js

```diff
@@ -0,0 +1,142 @@
+import SearchBar from './SearchBar/searchbar.js'
```

Comment:
Is this component still used ?
What's the difference with `src/components/Landing/landing.js` ?